web/src/components/Surfaces/ProfileImage.tsx

```diff
@@ -0,0 +1,71 @@
+import { ChangeEvent } from "react";
+import { MdAccountCircle, MdEdit } from "react-icons/md";
+
+import CircularProgress from "../Feedback/CircularProgress";
+import CustomImage from "./CustomImage";
+
+interface Props {
+  image: string | null;
+  onImageChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  isSelf: boolean;
+  isLoading: boolean;
+}
+
+export default function ProfileImage({
+  image,
+  onImageChange,
+  isSelf,
+  isLoading,
+}: Props) {
+  return (
+    <div className={"group relative"}>
+      {isSelf && (
+        // Allow user to update their profile image if they are viewing their own profile
+        <div
+          className={`absolute z-10 h-full w-full rounded-full transition-colors duration-200 ease-out group-hover:bg-slate-900/70
+                    ${isLoading ? "bg-slate-900/70" : ""}`}
+        >
+          {!isLoading ? (
+            <>
+              <input
+                type={"file"}
+                onChange={onImageChange}
+                title={" "}
+                aria-label={"Set Profile Image"}
+                className={
+                  "absolute z-10 h-full w-full  cursor-pointer opacity-0"
```

Comment:
Double space in this line (doesn't matter, but just noticed it)

Reply:
It is not longer there on the latest branch im working on